### PR TITLE
add evm networks for counterstake

### PR DESCRIPTION
**Pull request by Taump**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/counterstake exports TVL: 
        
 
 
    obyte                     184.71 k
    kava                      479.95
    ethereum                  307.38 k
    polygon                   4.87 k
    bsc                       1.32 k
    
    total                    498.76 k

**Comment by llamatester** [comment]
The adapter at projects/counterstake exports TVL: 
        
 
 
    obyte                     185.49 k
    kava                      480.92
    polygon                   3.05 k
    ethereum                  309.93 k
    bsc                       1.33 k
    
    total                    500.29 k

File: projects/counterstake/index.js [hunk 1]
```diff
@@ -72,78 +74,43 @@ async function totalObyteTvl(timestamp) {
...
-                }
-
+                tokensAndOwners.push([bridge.stake_asset, assistant_aa])
                 // imported asset
-                if (bridge.foreign_asset !== ZERO_ADDRESS) {
-                    erc20Contracts.push({ contract: assistant_aa, tokenContract: bridge.foreign_asset });
-                } else {
-                    nativeContracts.push({ contract: assistant_aa, tokenContract: bridge.foreign_asset });
-                }
+                // tokensAndOwners.push([bridge.foreign_asset, assistant_aa])
```
Comment (Taump) [inline projects/counterstake/index.js:102]: @g1nt0ki Hello, thanks for the refactoring. Why is it commented out?

Comment (g1nt0ki) [reply projects/counterstake/index.js:102]: we count the assets only on the source side, else it would be double counting.

**Pull request merged**